src/components/Checkbox/index.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import BaseCheckbox from './BaseCheckbox';
+
+const Checkbox = props => (
+    <BaseCheckbox
+        // eslint-disable-next-line react/jsx-props-no-spreading
+        {...props}
+        onMouseDown={props.onMouseDown ? props.onMouseDown : props.onPress}
```

Comment:
```suggestion
        onMouseDown={props.onMouseDown || props.onPress}
```

I might be wrong but this should be equivalent, for `.native `file too

Reply:
Let me just try to apply all these changes and test once